src/axiosOptions.js

```diff
@@ -1,16 +1,16 @@
 import axios from 'axios'
 
-const axiosGet = (url, params) =>
-  axios.get(url, { params, validateStatus: () => true })
+const axiosGet = (url, params, config) =>
+  axios.get(url, { params, ...config, validateStatus: () => true })
```

Comment:
Can you put params after config in the spread?

Otherwise it's not obvious for a consumer what happens if you include a `params` key within `config`.